src/openforms/forms/api/serializers/form.py

```diff
@@ -55,6 +55,45 @@ class FormLiteralsSerializer(serializers.Serializer):
     confirm_text = ButtonTextSerializer(raw_field="confirm_text", required=False)
 
 
+class FormAuthenticationBackendSerializer(serializers.ModelSerializer):
+    options = serializers.DictField(label=_("authentication backend options"))
+
+    class Meta:
+        model = FormAuthenticationBackend
+        fields = [
+            "backend",
+            "options",
+        ]
+
+    def get_fields(self):
+        # set backend choices at runtime
+        fields = super().get_fields()
+        fields["backend"].choices = auth_register.get_choices()
+        return fields
+
+    def validate(self, attrs):
+        # validate config options with selected plugin
+        if "backend" not in attrs:
+            # performing nested PATCH
+            return attrs
+        plugin = auth_register[attrs["backend"]]
+
+        if not plugin.configuration_options:
+            return attrs
+
+        serializer = plugin.configuration_options(
+            data=attrs["options"], context=self.context
+        )
+        try:
+            serializer.is_valid(raise_exception=True)
+        except serializers.ValidationError as e:
+            detail = {"options": e.detail}
+            raise serializers.ValidationError(detail) from e
```

Comment:
```suggestion
        if not serializer.is_valid():
            raise serializer.ValidationError({"options": serializer.errors})
```
for a simpler variant?